utils/createDiffText.js

```diff
@@ -0,0 +1,101 @@
+
+    if (property.includes("absoluteBoundingBox")) {
+      if (property.includes(".x") || property.includes(".y")) {
+        if (differenceTexts.Position) {
+          differenceTexts.Position = {
+            ...differenceTexts.Position,
+            [propertyParser(property)]: value,
+          };
+        } else {
+          differenceTexts.Position = {
+            [propertyParser(property)]: value,
+          };
+        }
+      } else if (property.includes(".width") || property.includes(".height")) {
+        if (differenceTexts.Size) {
+          differenceTexts.Size = {
+            ...differenceTexts.Size,
+            [propertyParser(property)]: value,
+          };
+        } else {
+          differenceTexts.Size = {
+            [propertyParser(property)]: value,
+          };
+        }
+      }
+    } else if (property.includes("characters") || property.includes("font")) {
+      if (differenceTexts.Text) {
+        differenceTexts.Text = {
+          ...differenceTexts.Text,
+          [propertyParser(property)]: value,
+        };
+      } else {
+        differenceTexts.Text = {
+          [propertyParser(property)]: value,
+        };
+      }
+    } else if (property.includes("Color") || property.includes("background")) {
+      if (differenceTexts.Color) {
+        differenceTexts.Color = {
+          ...differenceTexts.Color,
+          [propertyParser(property)]: value,
+        };
+      } else {
+        differenceTexts.Color = {
+          [propertyParser(property)]: value,
+        };
+      }
+    } else if (differenceTexts.Etc) {
+      differenceTexts.Etc = {
+        ...differenceTexts.Etc,
+        [propertyParser(property)]: value,
+      };
+    } else {
+      differenceTexts.Etc = {
+        [propertyParser(property)]: value,
+      };
+    }
+  }
+
+  let differenceResult = "";
+
+  for (const type in differenceTexts) {
+    const propValue = differenceTexts[type];
+
+    differenceResult += `\n  ${type}\n`;
+
+    for (const prop in propValue) {
+      const changeResult = propValue[prop];
+      const [beforeValue, afterValue] = changeResult.split("=>");
+
+      if (beforeValue === undefined && afterValue !== undefined) {
+        if (!isNaN(afterValue)) {
+          differenceResult += `   ${prop}: ${Number(afterValue).toFixed(1)} 추가됨.\n`;
+        } else {
+          differenceResult += `   ${prop}: ${afterValue} 추가됨.\n`;
+        }
+      } else if (beforeValue !== undefined && afterValue === undefined) {
+        if (!isNaN(beforeValue)) {
+          differenceResult += `   ${prop}: ${Number(beforeValue).toFixed(1)} 삭제됨.\n`;
+        } else {
+          differenceResult += `   ${prop}: ${beforeValue} 삭제됨.\n`;
+        }
+      } else if (!isNaN(beforeValue) && !isNaN(afterValue)) {
+        differenceResult += `   ${prop}: ${Number(beforeValue).toFixed(1)} => ${Number(afterValue).toFixed(1)}.\n`;
+      } else {
+        differenceResult += `   ${prop}: ${changeResult}.\n`;
```

Comment:
P1: 띄어쓰기 부분도 수정되면 좋을 것 같습니다 :)

```suggestion
        if (!isNaN(afterValue)) {
          differenceResult += `${prop}: ${Number(afterValue).toFixed(1)} 추가됨.\n`;
        } else {
          differenceResult += `${prop}: ${afterValue} 추가됨.\n`;
        }
      } else if (beforeValue !== undefined && afterValue === undefined) {
        if (!isNaN(beforeValue)) {
          differenceResult += `${prop}: ${Number(beforeValue).toFixed(1)} 삭제됨.\n`;
        } else {
          differenceResult += `${prop}: ${beforeValue} 삭제됨.\n`;
        }
      } else if (!isNaN(beforeValue) && !isNaN(afterValue)) {
        differenceResult += `${prop}: ${Number(beforeValue).toFixed(1)} => ${Number(afterValue).toFixed(1)}.\n`;
      } else {
        differenceResult += `${prop}: ${changeResult}.\n`;
```

Reply:
변경사항 text padding 값을 더 주기위해서 세부정보의 경우 2칸의 여백을 두었는데
텍스트 박스 자체 padding 값으로 렌더되도록 여백 삭제하겠습니다.